integration-cli/docker_cli_ps_test.go

```diff
@@ -508,3 +508,22 @@ func (s *DockerSuite) TestPsListContainersFilterCreated(c *check.C) {
 		c.Fatalf("Expected id %s, got %s for filter, out: %s", cID, containerOut, out)
 	}
 }
+
+func (s *DockerSuite) TestPsFormatMultiNames(c *check.C) {
```

Comment:
I looked and I couldn't find a test that used `--link` **and** didn't use `--no-trunc` - do you see one? If not it might be good to add one.